src/pages/messages/components/NviCandidateActions.tsx

```diff
@@ -191,8 +191,11 @@ export const NviCandidateActions = ({ nviCandidate, nviCandidateQueryKey }: NviC
                     date={note.date}
                     username={note.username}
                     backgroundColor="nvi.main"
+                    canDeleteMessage={user ? user.isNviCurator || user.nvaUsername === note.username : undefined}
```

Comment:
Basert på navnet ville jeg kanskje tenkt det er naturlig at man alltid sender inn en boolean her 🤔 `undefined` blir jo bare `false` uanz da, men syns det er mer tydelig når det er eksplisitt
```suggestion
                    canDeleteMessage={!!user && (user.isNviCurator || user.nvaUsername === note.username)}
```